src/kata2.booleanToWord.js

```diff
@@ -1,3 +1,5 @@
-const booleanToWord = (boolean) => {};
-
-module.exports = booleanToWord;
+const booleanToWord = (boolean) => {
+  if (boolean === true) return "yes";
+  if (boolean === false) return "no";
```

Comment:
for a boolean, you don't need to make an assertion like `=== true` because it's already true or false, i.e. can be written as `if (boolean)`

Reply:
because your first if statement returns a value you don't need the second if statement. if the boolean isn't true then it must be false. i.e. can be written as
```
if (boolean) return true;
return false;
```

Reply:
You can further simplify this logic by using the ternary operator e.g.
```
return boolean ? "Yes" : "No";
```

[You can read more about the ternary operator here](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Conditional_operator)